walk-in-interview/src/main/java/com/google/account/FirebaseAuthSessionCookieUtil.java

```diff
@@ -0,0 +1,137 @@
+package com.google.account;
+
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.firebase.FirebaseApp;
+import com.google.firebase.FirebaseOptions;
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.auth.FirebaseAuthException;
+import com.google.firebase.auth.FirebaseToken;
+import com.google.firebase.auth.SessionCookieOptions;
+import com.google.utils.ServletUtils;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.CookieParam;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.core.Cookie;
+import javax.ws.rs.core.NewCookie;
+import javax.ws.rs.core.Response;
+import java.io.IOException;
+import java.net.URI;
+import java.util.concurrent.TimeUnit;
+
+/** Util methods related to firebase auth session cookies. */
+public final class FirebaseAuthSessionCookieUtil {
+    private static final int SESSION_COOKIE_DURATION_DAYS = 5;
+    private static final String SESSION_COOKIE_NAME = "session";
+    private static final String ID_TOKEN_PARAM = "idToken";
+    private static final String LOG_IN_PAGE_PATH = "/log-in/index.html";
+
+    private static final String DATABASE_URL = "https://com-walk-in-interview.firebaseio.com/";
+    private static final String PROJECT_ID = "com-walk-in-interview";
+
+    public FirebaseAuthSessionCookieUtil() throws IOException {
+        initAdminSDK();
+    }
+
+    /**
+     * Creates the session cookies. It is evoked when the user signs in.
+     *
+     * @param request Log in request.
+     * @return Response with session cookie.
+     */
+    @POST
+    @Path("/business-log-in")
+    @Consumes("application/json")
+    public Response createSessionCookie(HttpServletRequest request) {
+        try {
+            // Gets the ID token sent by the client
+            String idToken = parseIDToken(request);
+
+            // Sets session expiration to 5 days
```

Comment:
nit, you should remove the "5 days" from your comment., it's defined by your constant. You can just say "set the expiration".